README.md

```diff
@@ -78,17 +109,35 @@ if (a) {
   }
 }
 ```
+//my solution
+
 
+if (a==true)
+	return 0
+else
+	return 2;
 
 ---
 
-**3)** _(Moderate, 20%)_
+**3)** _(Moderate, 20%)_ not done
 
 Write a function which returns a string containing the entire lyrics for the song "The Twelve Days of Christmas".  Make sure that your result is grammatically and typographically correct (include line breaks, commas, etc. where needed), but keep redundancy within your program to a minimum.  Don't just `console.log()` each line; return them together as a single string which includes a '\n' at each line break.
 You may want to use helper functions, loops, and/or arrays to store repeated elements.
 
 If you prefer a non-Christmas option, you may choose a different song with similarly repeating structure, such as "There was an Old Woman Who Swallowed a Fly".  If you prefer a vegan option, you may write your own cruelty-free song.
 
+
+//my solution for #3
+//
+var days = ["first", "second", "third", "fourth", "fifth", "sixth", "seventh", "eighth", "ninth", "tenth", "eleventh", "twelfth"];
+//count-down value
+var numberOfDays = days.length;
+
+var gifts = ["A Partridge in a Pear Tree", "Two Turtle Doves", "Three French Hens", "Four Calling Birds", "Five Golden Rings", "Six Geese a Laying", "Seven Swans a Swimming", "Eight Maids a Milking", "Nine Ladies Dancing", "Ten Lords a Leaping", "Eleven Pipers Piping", "12 Drummers Drumming"];
+
+for (var i = 0; i < numberOfDays; i++) {
+  console.log ("On the "+ days[i] + " day of Christmas my true love sent to me " + gifts [i]);
+}
 ---
```

Comment:
Great Start. I actually did the same thing when I had this assignment, but the song "twelve days of Christmas" adds the previous days gift to the end of the melody. "... three French hens, two turtle doves, and a partridge in a pear tree."

You need some kind of mechanism to cycle through the list backwards based on the day (or number of time you have been through the list).